pkg/admissioncontroller/webhooks/admission/internaldomainsecret/admission.go

```diff
@@ -74,15 +76,19 @@ func (h *handler) Handle(ctx context.Context, request admission.Request) admissi
 	if request.SubResource != "" {
 		return acadmission.Allowed("subresources on Secrets are not handled")
 	}
+	seedName, err := seedutils.ComputeSeedName(request.Namespace)
+	if request.Namespace != v1beta1constants.GardenNamespace && err != nil {
```

Comment:
Personally, I would have preferred to work with an empty string if the seed name cannot be computed from the namespace instead of now having to handle an error.